addons/mail/models/mail_message_schedule.py

```diff
@@ -64,10 +64,13 @@ def _send_notifications(self, default_notify_kwargs=None):
         for model, schedules in self._group_by_model().items():
             if model:
                 records = self.env[model].browse(schedules.mapped('mail_message_id.res_id'))
+                existing = records.exists()
             else:
                 records = [self.env['mail.thread']] * len(schedules)
 
             for record, schedule in zip(records, schedules):
+                if record not in existing:
+                    continue
```

Comment:
Should we log that a scheduled message could not be sent for model, res_id?

Reply:
Would say no, as anyway you won't be able to see it / manage it / ... tant pis.